tests/util/cw_util.go

```diff
@@ -0,0 +1,95 @@
+package util
+
+import (
+	"encoding/json"
+	"strconv"
+	"testing"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"gotest.tools/v3/assert"
+
+	"github.com/umee-network/umee/v5/client"
+)
+
+const (
+	SucceessRespCode = uint32(0)
+)
+
+type Cosmwasm struct {
+	StoreCode    uint64
+	ContractAddr string
+	Sender       string
+	T            *testing.T
+	umee         client.Client
+}
+
+func (cw *Cosmwasm) SetTestingF(t *testing.T) {
+	cw.T = t
+}
+
+func (cw *Cosmwasm) SetUmeeClient(c client.Client) {
```

Comment:
why do we have the `Set*` methods rather than constructor? I don't think `testing.T` nor `cw.umee` should ever change once a `Cosmwasm` contract is created.